dist/obsworker

```diff
@@ -74,7 +74,7 @@ if [ -n "$OBS_WORKER_WIPE_AFTER_BUILD" ]; then
     OBS_WIPE_AFTER_BUILD="--wipeafterbuild"
 fi
 
-if [ -n "OBS_WORKER_SECURITY_LEVEL" ]; then
+if [ -n "$OBS_WORKER_SECURITY_LEVEL" ]; then
```

Comment:
@dmarcoux that's the kind issue that ShellCheck would catch, therefore the PR https://github.com/openSUSE/open-build-service/pull/9388 should be merged

Reply:
I agree that ShellCheck is useful. However, it's only targeting one file in that PR. It's better than nothing, but still quite limited.